recipes/fido2/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "fido2" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4483b48ab3da0a3f3cc8e2f11d833160dfa2e6d12148e938982df4acd10b2682
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
```

Comment:
Don't need the `build` section under requirements in this recipe. Just using the `host` section should be fine.

Reply:
i remove the build section